spec/support/controller_macros.rb

```diff
@@ -2,14 +2,14 @@ module ControllerMacros
   def login_admin
     before(:each) do
       user = FactoryGirl.create(:admin)
-      @request.env["devise.mapping"] = Devise.mappings[:user]
+      @request.env['devise.mapping'] = Devise.mappings[:user]
       sign_in user
     end
   end
-  
+
   def login_curator
     before(:each) do
-      @request.env["devise.mapping"] = Devise.mappings[:user]
+      @request.env['devise.mapping'] = Devise.mappings[:user]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.